test/scripts/base-host-check.sh

```diff
@@ -78,6 +79,22 @@ get_oscap_score() {
     fi
 }
 
+check_ca_cert() {
+    serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2-)
+
+    echo "📗 Checking CA cert anchor file"
+    if ! [ -e "/etc/pki/ca-trust/source/anchors/${serial}.pem" ]; then
+        echo "Anchor CA file does not exist"
+        exit 1
+    fi
+
+    echo "📗 Checking extracted CA cert file"
+    if ! [ -e "/etc/pki/ca-trust/source/extracted/pem/directory-hash/Test_CA_for_osbuild.pem.pem" ]; then
+        echo "Extracted CA file does not exist"
+        exit 1
+    fi
+}
+
```

Comment:
If the manifest were not rebuilt during the CI run (because it didn't change), you'd find a link to the run in which it was tested in the job that generated it (e.g. https://gitlab.com/redhat/services/products/image-builder/ci/images/-/jobs/8565731980). Since the manifest didn't change and you added the test, it was not run at all. A complete manifest regeneration would be required. That can be achieved by bumping the `rngseed` in https://github.com/osbuild/images/blob/048c8c6745068f7a91c8fe48422b64d37d34c5ff/Schutzfile#L3

I did it in https://github.com/osbuild/images/pull/1093 and if my comment about the potential typo from above is correct, it should fail :thinking:

Reply:
The story continuous:

- https://github.com/osbuild/images/pull/907 added the `cacert` customization only to `all-customizations.json`,
- `all-customizations.json` is applied only when building `qcow2` https://github.com/osbuild/images/blob/048c8c6745068f7a91c8fe48422b64d37d34c5ff/test/config-map.json#L2-L13
- We do not boot-test `qcow2` in this repository CI, so the change in the `base-host-check.sh` won't run in the CI.

Reply:
Yeah, apologies: https://github.com/osbuild/images/pull/1096